### DISCUSSION TITLE
how to add the windhawk mods?

### DISCUSSION THREAD

**Discussion by thekushdhingra** [ashish0kumar/windots#17] [Issues & Help] [answered]
.

**Comment by ashish0kumar** [comment, marked as answer]
- Download and Install [Windhawk](https://windhawk.net)
- Open Windhawk and go to the "**Explore**" section. Search for and install the following mods:
    - Windows 11 Start Menu Styler
    - Windows 11 Taskbar Styler
    - Windows 11 Notification Center Styler

- **Optional**: If you want to apply my config
    - Go to the "**Advanced**" section of each installed mod.
    - Paste [my custom configs](https://github.com/ashish0kumar/windots/tree/main/.config/windhawk) into the "Mod Settings" field for each mod.
    
Let me know if you need any further help.
